dev-python/hpack/hpack-3.0.0.ebuild

```diff
@@ -34,6 +34,11 @@ python_prepare_all() {
 	# Remove a test that is not part of the mainstream tests
 	# Also, it's data directory is not included in the release
 	rm test/test_hpack_integration.py || die
+
+	# depends on old enum representation
+	sed -e 's:test_get_by_index_out_of_range:_&:' \
```

Comment:
We don't do seds on package using pytest anymore, we use `--deselect` instead. Done that.